BUMP_VERSION.asciidoc

```diff
@@ -0,0 +1,196 @@
+be done to bump to the next version of Elasticsearch.  Note that the same procedure is followed
+when bumping the bug-fix version (for example, after releasing 5.2.1, bumping the release version
+to 5.2.2).
+
+==== Upgrade the version
+
+Since 5.2.0 has been released, the next bug-fix version in the 5.2.0 line 
+will be 5.2.1.  Create a V_5_2_1 version constant in `core/src/main/java/org/elasticsearch/Version.java`
+for all branches >= 5.2.  In this case, the V_5_2_1 version constant should go into
+the 5.2, 5.x, and master branches.  
+
+For the 5.2 branch only, set the CURRENT version in `Version.java` to the new V_5_2_1 constant.  
+For the 5.x and master branches, just add the V_5_2_1 constant without setting the CURRENT version.
+-------------------------------------
+  + public static final int V_5_2_1_ID_UNRELEASED = 5020199;
+  + public static final Version V_5_2_1_UNRELEASED = new Version(V_5_2_1_ID, org.apache.lucene.util.Version.LUCENE_6_3_0);
+  + public static final Version CURRENT = V_5_2_1_UNRELEASED;
+-------------------------------------
+
+Also, update the `Version#fromId` method to include the new 5.2.1 version in each branch
+-------------------------------------
+   public static Version fromId(int id) {
+       switch (id) {
+  +       case V_5_2_1_ID_UNRELEASED:
+  +           return V_5_2_1_UNRELEASED;
+-------------------------------------
+
+==== Upgrade version.properties
+
+In the 5.2 branch, change the elasticsearch version in `buildSrc/version.properties` to 5.2.1:
+-------------------------------------
+  -elasticsearch     = 5.2.0
+  +elasticsearch     = 5.2.1
+-------------------------------------
+
+==== Add the version to the vagrant tests
+
+In the `qa/vagrant/versions` file, add the newly released 5.2.0:
+-------------------------------------
+  +5.2.0
+-------------------------------------
+
+==== Generate the backward compatibility indices
+
+Generate the backward compatibility (bwc) indices for the bwc tests for the just released 5.2.0 
+with the following steps:
+
+- Make sure python3 is installed, if you don't already have it (on a Mac, execute `brew install python3`)
+- Make sure the Elasticsearch Python client is installed, if you don't have it already.
+-------------------------------------
+  pip install elasticsearch
+-------------------------------------
+- You may need to additionally install the requests package in python3:
+-------------------------------------
+  pip3 install requests
+-------------------------------------
+- Change directories to the `dev-tools` directory under the root Elasticsearch repository.
+- Get the distribution of the newly released Elasticsearch version (5.2.0 in our scenario) and
+  untar it to the `backwards` folder inside `dev-tools`:
+-------------------------------------
+  mkdir -p {path_to_repo}/dev-tools/backwards && tar -zxvf {path_to_distribution}/elasticsearch-5.2.0.tar.gz -C {path_to_repo}/dev-tools/backwards
```

Comment:
You're right, i tried it again, its because i was running the tool from within dev-tools that i had to set up the rest that way.  I can make this simpler by running in the root folder.  Thanks for pointing that out